redisinsight/api/src/modules/cloud/subscription/models/capi.interface.ts

```diff
@@ -49,3 +49,18 @@ export interface ICloudCapiSubscriptionPlan {
   provider: string;
   region: string;
 }
+
+export interface ICloudCapiSubscriptionCloudRegion {
```

Comment:
please, create api.interface file and put this interface with proper name there

Reply:
renamed